Adamant/Services/FilesNetworkManager/IPFSApiCore.swift

```diff
@@ -34,14 +34,34 @@ extension IPFSApiCore: BlockchainHealthCheckableService {
         let statusResponse = await getNodeStatus(origin: origin)
         let ping = Date.now.timeIntervalSince1970 - startTimestamp
         
-        return statusResponse.map { _ in
+        return statusResponse.map {
             .init(
                 ping: ping,
-                height: .zero,
+                height: getHeightFrom(timestamp: $0.timestamp),
                 wsEnabled: false,
                 wsPort: nil,
-                version: nil
+                version: getVersionFrom(stringVersion: $0.version)
             )
         }
     }
+    
+    private func getHeightFrom(timestamp: UInt64) -> Int {
+        let timestampInSeconds = String(timestamp / 1000)
+        
+        /// Chicking the len of string representation of a timestamp in seconds
+        /// to cut first two symbols from it and convert it to height
+        if timestampInSeconds.count >= 2 {
+            let subString = timestampInSeconds[
+                timestampInSeconds.index(timestampInSeconds.startIndex, offsetBy: 2)..<timestampInSeconds.endIndex
+            ]
+            let string = String(subString)
+            return Int(string) ?? .zero
+        } else {
+            return .zero
+        }
+    }
+    
+    private func getVersionFrom(stringVersion: String) -> Version? {
+        .init(stringVersion)
+    }
 }
```

Comment:
We use `private extension` in the project for defining private methods. There will be no need to write `private` in every function